apis/config/v1alpha2/configuration_types.go

```diff
@@ -45,6 +45,29 @@ type Configuration struct {
 
 	// InternalCertManagement is configuration for internalCertManagement
 	InternalCertManagement *InternalCertManagement `json:"internalCertManagement,omitempty"`
+
+	// WaitForPodsReady is configuration for waitForPodsReady
+	WaitForPodsReady *WaitForPodsReady `json:"waitForPodsReady,omitempty"`
+}
+
+type WaitForPodsReady struct {
```

Comment:
Separate PR?

Reply:
Isn't this PR for the KEP? we shouldn't have implementation changes here.

Reply:
This is a proposal for the API changes in Design Details. The implementation itself will be a separate PR.

Reply:
oh, I see, this was commited by mistake, reverting this file